browser-test/src/support/applicant_program_list.ts

```diff
@@ -0,0 +1,139 @@
+import {Page, Locator} from '@playwright/test'
+
+/**
+ * Class for interacting with the applicant program list page
```

Comment:
**thought (non-blocking):** We have so many names for this page.  "Landing page", "homepage", "program index page" and "applicant program list page".  It would be nice if we could start using one name for it.  I'm not a fan of "program index" but, since that's the name of the view and the template, maybe it would be helpful to name this support file accordingly?  I don't feel strongly about this.  Let me know what you think.

Reply:
I'm going to defer renaming this. Let's check with Sijia and decide on one name.

It doesn't feel like a landing page to me, nor a home page as you get different views depending on who you are logged in as.